themes/UPDATING-NODE-MODULES.md

```diff
@@ -1,26 +1,30 @@
-## Updating dependencies for login, admin console, and old account console
+# Updating dependencies
```

Comment:
I would like to see a solution to updating dependencies that doesn't require installing npm.

Reply:
@stianst not sure if I understand what you're looking for. But npm is the standard package manager for Node.js, as you already know. Another alternative is to use yarn (another package manager), but still, it's necessary some tooling in place.

Reply:
I think what Stian means is that he'd like to be able to run NPM related tasks trough Maven, so that he does not have to set up this tooling himself.

@stianst is this something that you want as part of this PR, or can we log an issue? Considering we might be moving this out of the repo into it's own (mono)repo we might do the rest of the work there instead.

Reply:
Since we've agreed that this will be moved to a separate "JS" repository in the future I'm okay with the current approach in this PR